### ISSUE TITLE
React.PropTypes has been depricated

### ISSUE THREAD

**Issue by Thomas101** [romainberger/react-portal-tooltip#42]
React has deprecated `React.PropTypes` in favour of using the `prop-types` repository. Suggest changing...

```js
import React, {PropTypes} from 'react'
```
to
```js
import React from 'react'
import PropTypes from 'prop-types'
```

Happy to open a PR if it makes life easier :)

**Comment by basicdays** [comment]
Looks like PR #43 would close out this issue. Is there a release plan to test/publish the fix possibly?

**Comment by romainberger** [comment]
The PR has been merged in version v1.2.0. Thank you and sorry for the delay!

**Comment by Thomas101** [comment]
Awesome thanks so much :)
